portfolio/src/main/webapp/script.js

```diff
@@ -85,3 +81,26 @@ function convertTime(timestamp) {
     return formattedTime;
 }
 
+/**
+ * This function is to get data from "/data" Servlet and fetch the latest data every 3 seconds 
+ */
+
+var getting = {
+
+        url:'/data',
+        dataType:'json',
+        success:function(res) {
+
+        const commentsElement = document.getElementById('comment-list');
+        commentsElement.innerHTML = '';
+        const commentListElement = document.createElement('ul');
+
+        for (comment of res) {
+            commentListElement.appendChild(createListElement(comment));
+        }
+        commentsElement.appendChild(commentListElement);
+}
+};
+
+window.setInterval(function(){$.ajax(getting)},3000);
```

Comment:
Is it possible to use getDataFromDataServlet as the first parameter in setInterval?

Reply:
I tried to use the getDataFromDataServlet function as the first parameter, but it will responds with some error message, I suppose I should follow some ajax format. Not quite familiar with javascript and ajax, so I just follow the most basic format.

Reply:
I see. I am not familiar with Javascript either. Maybe you can put the error message in our team's group chat, and if someone happens to have experience in this, he/she can offer some insights.